style.css

```
@@ -81,3 +86,214 @@ a {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 0 16px;
+}
+
+.icons2 {
+  display: flex;
+  flex-direction: row;
+  padding-left: 16px;
+}
+
+.icons2 li {
+  text-align: center;
+  list-style: none;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.btn {
+  padding: 10px;
+  border: 1px solid #6070ff;
+  border-radius: 10px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  background-color: #fff;
+  margin: 0 26px;
+}
+
+.description {
+  display: flex;
+  margin: 0 16px;
+  align-items: center;
+  padding-left: 8px;
+}
+
+.snip1 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.snip2 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.snip3 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.title-about {
+  font-family: Poppins, sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 52px;
+  text-align: left;
+  padding: 158px 24px 0 24px;
+}
+
+.para-about {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: 0 24px 0 24px;
+}
+
+.card-w2 {
+  background: #fff;
+  border-radius: 20px;
+  padding-bottom: 20px;
+  margin: 0 24px;
+}
+
+.card-w3 {
+  background: #fff;
+  border-radius: 20px;
+  padding-bottom: 20px;
+  margin: 0 24px;
+}
+
+.card-w4 {
+  background: #fff;
+  border-radius: 20px;
+  padding-bottom: 20px;
+  margin: 0 24px;
+}
+
+.open-menu {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  align-items: baseline;
+  background-color: #f7f7f9;
+  border-radius: 10px;
+  margin: 18px 12px 0 12px;
+  padding: 8px 0 8px 5px;
+}
+
+.open-menu p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  align-items: center;
+  color: #253858;
+}
+
+.main-menu {
+  display: flex;
+  justify-content: space-around;
+}
+
+.main-menu p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  align-items: center;
+  color: #000;
+  margin-left: 16px;
+}
+
+.arrow {
+  padding-left: 64px;
+  margin-left: 64px;
+}
+
+.arrow1 {
+  padding-left: 64px;
+  margin-left: 64px;
+}
+
+.arrow2 {
+  padding-left: 64px;
+  margin-left: 64px;
+}
```

[OPTIONAL]

- [ ] You should always follow the DRY principle. These three classes exactly the same. So this means two of them are unnecessary. (Multiple HTML elements can share the same class.)

If you make this suggested change, please remember the update classes of HTML elements also.

You can [check this video](https://www.youtube.com/watch?v=0px6YH-cauQ) for more information. 

> Don't repeat yourself (DRY, or sometimes do not repeat yourself) is a principle of software development aimed at reducing repetition of software patterns, replacing them with abstractions, or using data normalization to avoid redundancy. 

```suggestion
.arrow {
  padding-left: 64px;
  margin-left: 64px;
}
```